docs/pages/user-guides.mdx

```diff
@@ -0,0 +1,225 @@
+Enter password for Teleport user alice:
+Tap any security key
+> Profile URL:        https://teleport.example.com:443
+  Logged in as:       alice
+  Cluster:            example.com
+  Roles:              access
+  Logins:             root, ubuntu, ec2-user
+  Kubernetes:         enabled
+  Valid until:        2022-11-01 22:37:05 -0500 CDT [valid for 12h0m0s]
+  Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty, private-key-policy
+```
+
+</ScopedBlock>
+
+<ScopedBlock scope={["cloud"]}>
+
+```code
+$ tsh login --proxy=mytennant.teleport.sh --user=alice
+Enter password for Teleport user alice:
+Tap any security key
+> Profile URL:        https://teleport.example.com:443
+  Logged in as:       alice
+  Cluster:            mytennant.teleport.sh
+  Roles:              access
+  Logins:             root, ubuntu, ec2-user
+  Kubernetes:         enabled
+  Valid until:        2022-11-01 22:37:05 -0500 CDT [valid for 12h0m0s]
+  Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty, private-key-policy
+```
+
+</ScopedBlock>
+
+Depending on how Teleport was configured for your network, you may need
+additional flags, such as `--auth=a_saml_connector`. Your administrator should
+provde the details required for your particular use case.
+
+## Teleport Connect
+
+The Teleport Connect app provides all the same access to resources as `tsh` and
+more, in a friendly graphic user interface. After
+[downloading](https://goteleport.com/download/) and installing Teleport Connect,
+you can log in initiate sessions for Server, Database, and Desktop Access all
+within a single window.
+
+1. Click **CONNECT** to connect to the Teleport plane:
+
+   ![A new instance of Teleport Connect](../img/teleport-connect-init.png)
+
+2. Provide the address of your Teleport Cluster (e.g. 
+<ScopedBlock scope={["oss", "enterprise"]}>`https://teleport.example.com`</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>`https://mytennant.teleport.sh`</ScopedBlock>
+) and click **NEXT**.
+
+3. Teleport Connect will ask you for your username, password, and MFA. If this
+Teleport plane is integrated with an external Identity Provider (**IdP**), it
+may instead prompt you to authenticate in a browser window with that service.
+
+4. Once authenticated, you can browse and connect to all the resources your
+user is permitted to access:
+
+   ![An example of Teleport Connect, populated with instances](../img/use-teleport/connect-ui-overview.png)
+
+## Server Access
+
+<Tabs>
+<TabItem label="tsh">
+
+`tsh ls` lists the servers you have access to through Server Access (ssh):
+
+```code
+$ tsh ls
+Node Name           Address         Labels                       
+------------------- --------------- ---------------------------- 
+server1.example.com 192.0.2.24:3022 access=servers,hostname=server1
+server2.example.com 192.0.2.32:3022 access=servers,hostname=server2
+```
+
+To connect to a server:
+
+```code
+$tsh ssh server1.example.com
```

Comment:
It would likely be more useful for end users to have the username included e.g. `tsh ssh ubuntu@server1.example.com`

Reply:
Adjusted.